back-office/shop-parameters/customer-settings/groups.md

```diff
@@ -0,0 +1,141 @@
+
+**Show prices.** By default, all **customers can view the prices**. If set to 'No', it activates the catalog mode for this group of customers: they should not be able to get the prices when browsing the front office nor have any shopping cart. It also impacts the customer account on the front office, turning off the 'Order history and details', 'Credit slips', and 'Merchandise returns' pages. A help text is available: `Need to hide prices for all groups? Save time, enable catalog mode in Product Settings instead.` in `Admin.Shopparameters.Help`.
+
+
+## Edit form
+
+Users can also set per-category and per-module settings. Once the customers group saved, clicking 'Edit' in the groups listing should load the same form with 2 extra options:
+
+### Category discount
+
+Merchants can **set a discount per-category for this given group of customers**. Clicking 'Add a category discount' displays a modal window entitled 'New group category discount' that shows the categories arborescence of the shop, clickable through radio buttons. Only one category can be selected.
+
+Two buttons at the top right-hand corner allow to 'Expand all' or 'Collapse all'. Clicking on 'Expand all' deploys all categories and subcategories and turns this button into 'Collapse all'. Clicking 'Collapse all' displays only the parent category, 'Home' by default.
+
+Only products that have the selected category as their default category should be affected by the discount. The expected behavior of a category discount works as followed: it prevails over any discount a member of this group could benefit from in this category. See notifications below.
+
+An information notification id displayed: `Caution: The discount applied to a category does not stack with the overall reduction but instead replaces it.` in `Admin.Shopparameters.Notification`. Right below is a warning notification displayed: `Only products that have this category as the default category will be affected.` in `Admin.Shopparameters.Notification`.
+
+**Discount (%).** Only numbers can be typed in the field. If the merchant tries to save letters or a negative value, then he should stay on this modal window and get an error notification right below the field: `The discount value is incorrect (must be a percentage).` in `Admin.Shopparameters.Notification`.
+
+Clicking 'Add' saves preferences. It displays the category discount(s) in the 'Edit' form, just below the 'Category discount' field, with the following information: route to the category, discount value, and a 'Delete' button. If a user deletes a discount, the line simply disappears.
+
+
+### Authorized modules
+
+With this setting, **merchants can authorize or prevent members of this group to access and use some of the shop's modules**. A list of all the store's enabled modules is displayed with a switch button to enable or disable each module. By default, all modules are enabled.
```

Comment:
```suggestion
With this setting, **merchants can authorize or prevent members of this group to access and use some of the shop's modules**. A list of all the store's installed modules is displayed with a switch button to enable or disable each module. By default, all modules are enabled.
```
It seems it's all the installed modules, no matter if they are enabled or disabled